moto/core/models.py

```diff
@@ -715,6 +715,27 @@ def get_config_resource(
         """
         raise NotImplementedError()
 
+    def aggregate_regions(self, path, backend_region, resource_region):
+        """
+        Returns a list of "region\1eresourcename" strings
```

Comment:
What is the `1e`?

Reply:
Update the comments to be clear that this is to be used for both aggregated and non-aggregated calls. It will figure out how to return the full list of resources for a given regional backend.

I'm going to assume that this _doesn't_ work for global types (or the odd partial global types, like S3)?

Reply:
`\1e` was actually supposed to be `\x1e` which is the ascii record separator. I changed it. I also updated the comments to make it clear.